partitura/io/importmei.py

```diff
@@ -68,6 +70,11 @@ def __init__(self, mei_path: PathLike) -> None:
         self.parts = (
             None  # parts get initialized in create_parts() and filled in fill_parts()
         )
+        # find the music tag inside the document
```

Comment:
```suggestion
        # find the music tag inside the document
        music_el = self.document.findall(self._ns_name("music", all=True))
        if len(music_el) != 1:
            raise Exception("Only MEI with a single music element are supported")
        self.music_el = music_el[0]
```
What is a single music element, do you mean at least one note?

Reply:
Music element refer to the XML element called music.

Reply:
added <music> for better clarity